source/api_ref.html.md.erb

```diff
@@ -1,6 +1,6 @@
 ---
-title: "API reference"
+title: "Incorporate an OpenAPI spec"
```

Comment:
How about "Add an OpenAPI specification to your documentation"?

Reply:
That's much better. Thank you. Will reword.